metadata_manager.py

```diff
@@ -5,7 +5,7 @@
 class MetadataManager:
 
     def __init__(self):
-        self.conn = sqlite3.connect('metadata.db')
+        self.conn = sqlite3.connect('metadata.db', check_same_thread=False)
```

Comment:
sqlite3 was giving me errors yesterday saying that the connection are not created from the same thread. I don't know what happened there, so I added the flag so that it doesn't complain about the object being created from different threads. (It might be a bad workaround though...)